sendgrid_sender.go

```diff
@@ -73,6 +73,10 @@ func buildMail(m mail.Message) (*smail.SGMailV3, error) {
 		p.AddTos(smail.NewEmail(to.Name, to.Address))
 	}
 
+	for k, v := range m.Data {
+		p.SetCustomArg(k, v.(string))
```

Comment:
Very close! we now need to use just one of the keys in that m.Data for this purpose. Also, we need to add documentation in our README that explains how to add custom args.

Reply:
Got it!